assets/js/popover/search.js

```diff
@@ -32,6 +32,6 @@ module.exports = function(context, cb) {
       return cb(err);
     }
     displayResults(search, documents, totalCount);
-    cb(null);
```

Comment:
And regarding your question, it's used by https://github.com/AnyFetch/anyfetch-chrome/pull/66/files#diff-0e3402195afcadb050404f9ab6de8e09R90